tests/test_launcher.py

```diff
@@ -391,11 +397,11 @@ def test_build_journal_argument(topy, journal_file_names, result, raises):
         assert _build_journal_argument(topy, journal_file_names) == result
 
 
+@pytest.mark.standalone
```

Comment:
```suggestion
```
I don't think this one should be marked as `standalone`, it should run with containers as well, and it is testing the deprecated warning message which is applicable to all launchers

Reply:
Done. Thank you.